### DISCUSSION TITLE
Compatility with Android SDK 20

### DISCUSSION THREAD

**Discussion by 1n3JgKl9pQ6cUMrW** [admob-plus/admob-plus#312] [Q&A]
Version `1.8.0` of `Admob Plus Cordova Android` updates to `SDK 20`.

This SDK has many breaking changes, compared with `SDK 19` and below.

https://developers.google.com/admob/android/migration

---

The most relevant breaking changes are the deprecation of the `Smart Banner` size in favor of the `Adaptive Banner`.

https://developers.google.com/admob/android/migration#smart

`1.8.0` does not have (native) support for the Adaptive Banner and still relies on the Smart Banner.

---

The `RewardedVideoAd` is deprecated about 18 months ago and totally removed from SDK 20

https://developers.google.com/admob/android/migration#removed

**How does version `1.8.0` handle those changes and will it be fully compatible in the future with version 20?**

**Comment by ratson** [comment]
@1n3JgKl9pQ6cUMrW Since AdMob Plus does not use any of the legacy API, upgrading to `SDK 20` only introduce the new deprecation warnings. Those deprecation usage will be address in future releases, some may require major version bump.

Not in hurry to make breaking changes as the `SDK 20` is still new and more deprecation and new APIs maybe coming.
